src/Annotation/Access.php

```diff
@@ -16,8 +16,6 @@ final class Access implements Annotation
      * Field access.
      *
      * @required
-     *
-     * @var string
      */
-    public $value;
+    public string $value;
```

Comment:
`@var` should not be removed as there are used by Doctrine Annotation for validation. See https://www.doctrine-project.org/projects/doctrine-annotations/en/1.10/custom.html#attribute-types
The properties can still be type hinted.

Reply:
Are you sure? Because all tests pass. The documentation could be obsolete

Reply:
Type-hinted properties cannot be invalid 🙂

Reply:
So, I checked but if we remove the `@var`, Doctrine doesn't do the check anymore and it doesn't use the type hint.
If we remove it and add the type hint, it will work in most of the case but if it is invalid, we get an error like this:
`Typed property Overblog\GraphQLBundle\Annotation\Access::$value must be float, string used`
instead of an error like this:
`[Type Error] Attribute "value" of @GQL\Access declared on property App\Entity\School::$notes expects
   a(n) string, but got boolean.`

For me the second one is more clear. Also, doctrine is able to validate more complexe type like `@var string[]` and we can't check that with type hint.

I think the tests still pass cause we are not testing invalid annotations.

Reply:
Makes sense. But I didn't remove `@var` annotation for arrays.

Reply:
So are we sure we want to type hint the annotations properties ? 
If we make a type mistake in an annotation, we will get a less explicit errors as the `Typed property` error will be raised instead of the usual annotation error.

This kind of message `Typed property Overblog\GraphQLBundle\Annotation\Access::$value must be float, string used` doesn't really help to locate the problem.

Reply:
And usually, end users do not manipulate this annotation classes.

Reply:
> If we make a type mistake in an annotation, we will get a less explicit errors as the Typed property error will be raised instead of the usual annotation error.

Not true. Before data is actually stored in the annotation class it's first parsed and validated by AnnotationReader, therefore validation will be performed first.

Reply:
Oh so sorry then. I tested it quick but I guess I had removed the `@var` before testing.